modules/velocyto.nf

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env nextflow
+
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+*/
+
```

Comment:
There seems to be a much easier-to-use tool called scvelo which we should consider switching to. It's actually a generalization of the principle's behind velocyto's RNA velocity estimation, but it seems much faster and easier to use.